docs/reference/index.rst

```diff
@@ -1,15 +1,22 @@
+.. py:currentmodule:: ppb
+
 API Reference
 ================
 
 For as simple as the tutorials make ``ppb`` look there's a lot of power under
 the hood. This section will cover the raw what of the ppb API. To find out why
-decisions are made, see the discussion section.
+decisions are made, see the :doc:`/discussion/index` section.
+
+.. automodule:: ppb
+
+.. autofunction:: run
```

Comment:
Did you build the docs locally? I think these two should be indented.

Reply:
Yup, looked fine to me? Automodule just prints out the docstring as text, it doesn't give a heading the same way `.. autoclass` or `.. autofunction` do.

Give me a second and I'll post a screenshot.